.yarn/versions/9dcc50ee.yml

```diff
@@ -0,0 +1,11 @@
+releases:
+  "@near-eth/aurora-ether": patch
+  "@near-eth/near-ether": patch
+
+
+declined:
+  - "@near-eth/aurora-erc20"
+  - "@near-eth/aurora-nep141"
+  - "@near-eth/nep141-erc20"
+  - "@near-eth/rainbow"
+  - rainbow-bridge-client-monorepo
```

Comment:
`@near-eth/rainbow` and `rainbow-bridge-client-monorepo` both need to be released also.

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge-client/pull/97/commits/1d4692168a4ac79abec89ae49ac3ed828307b811